### ISSUE TITLE
Verify against design spec

### ISSUE THREAD

**Issue by jbphet** [phetsims/states-of-matter#31]
ActualConcepts has delivered a version of this simulation that is ready to be verified against the design spec.  This will be assigned to the original designer of the HTML5 version.

Link to version that should be verified: http://www.colorado.edu/physics/phet/dev/html/states-of-matter/1.0.0-dev.3/states-of-matter_en.html.

**Comment by jbphet** [comment]
@arouinfar - Please log anything that needs to be resolved in a separate issue, and close this issue once your review is complete.

**Comment by arouinfar** [comment]
Review complete. Closing.
